Product/Production/Gateway/DocumentQuery_20/src/main/resources/docquery/_20/webservices.xml

```diff
@@ -57,6 +60,9 @@ http://cxf.apache.org/core http://cxf.apache.org/schemas/core.xsd ">
         <jaxws:handlers>
             <ref bean="TransactionHandler" />
         </jaxws:handlers>
+        <jaxws:outInterceptors>
```

Comment:
This isn't right.  Needs to only be on the NHIN endpoint.  Need to do that for all 3 services (only one endpoint in each file).